entrypoint.sh

```diff
@@ -2,6 +2,9 @@
 set -e
 
 if ! grep ${SSH_USER} /etc/passwd ; then
+    # Install dependencies
+    apt-get install -y ${SSH_DEPENDENCIES}
```

Comment:
@ruiztulio 
The environment variable `SSH_DEPENDENCIES` it allows to install another package.
For example
`SSH_DEPENDENCIES=git` for to install `git`